### PR TITLE
transform

### PR DESCRIPTION
**Pull request by MinchukO**

### CODE REVIEW

File: src/styles/block/product-card.scss [hunk 1]
```diff
@@ -10,6 +10,13 @@
     width: 160px;
     height: 134px;
     margin: 0 auto 45px;
+    transition-duration: 400ms;
+    &:hover {
+      transform: scale(1.2);
+      outline: 3px solid #f3f3f3;
+      padding: 5px 0;
+      transition: border 400ms easy 0s;
```
Comment (vadym-liubych) [inline src/styles/block/product-card.scss]: лишняя строчка

File: src/styles/normalize.scss [hunk 2]
```diff
@@ -4,3 +4,7 @@ html {
   line-height: 13px;
   font-weight: 400;
 }
+
+a {
+  transition-duration: 400ms;
```
Comment (vadym-liubych) [inline src/styles/normalize.scss]: Замени на  transition: color 0.4s, background-color 0.4s;

**Review by vadym-liubych — APPROVED** [review]
Good job!

File: src/styles/block/product-card.scss [hunk 3]
```diff
@@ -10,6 +10,12 @@
     width: 160px;
     height: 134px;
     margin: 0 auto 45px;
+    transition-duration: 400ms;
```
Comment (aliceblunt8) [inline src/styles/block/product-card.scss:13]: add transition for transform property, now it's for all properties

File: src/styles/normalize.scss [hunk 4]
```diff
@@ -4,3 +4,7 @@ html {
   line-height: 13px;
   font-weight: 400;
 }
+
+a {
+  transition: color 0.4s, background-color 0.4s;
+}
```
Comment (aliceblunt8) [inline src/styles/normalize.scss:10]: it's better to add transition to specific elements (links and buttons) using classes, not to all anchor tags

**Review by aliceblunt8 — COMMENTED** [review]
good job!
remove either width or height from image, cause your image looks stretched on scaling

image

**Pull request merged**